lints.toml

```diff
@@ -39,14 +39,18 @@ allow = [
     'clippy::match_wildcard_for_single_variants',
     'clippy::missing_docs_in_private_items',
     'clippy::missing_inline_in_public_items',
+    'clippy::module_name_repetitions',
     'clippy::must_use_candidate',
+    'clippy::panic',
     'clippy::pattern_type_mismatch',
     'clippy::semicolon_if_nothing_returned',
     'clippy::shadow_reuse',
     'clippy::shadow_same',
     'clippy::unreachable',
+    'clippy::expect_used',
+    'clippy::unwrap_used',
```

Comment:
We should explicitly allow them and unwrap only in tests. We should `expect` in all the conditions not in tests.

Reply:
Therefore I don't think we should add them here.